pkg/resourcecreator/google/sql/instance.go

```diff
@@ -66,6 +67,10 @@ func GoogleSqlInstance(objectMeta metav1.ObjectMeta, instance nais_io_v1.CloudSq
 
 	flags := []google_sql_crd.SQLDatabaseFlag{{Name: "cloudsql.iam_authentication", Value: "on"}}
 	for _, flag := range instance.Flags {
+		valid, err := ValidateFlag(flag.Name, flag.Value)
+		if !valid || err != nil {
+			log.Errorf("sql instance flag '%s' is not valid: %v", flag.Name, err)
```

Comment:
Dette vil bare logge det et sted vi kan se, burde det føre til en feil som treffer brukeren mer direkte?

Reply:
Hvis vi feiler hardt, så får vi plutselig ansvaret for at denne lista over valideringer er synket med Cloud SQL.
Det føles feil, vi har ingen kontroll på når Google bestemmer seg for å endre dette.
Det beste alternativet blir kanskje å lage en Event?